.github/workflows/serverless.yaml

```diff
@@ -0,0 +1,35 @@
+name: Serverless
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - name: Install scylla-ccm
+        run: pip3 install https://github.com/scylladb/scylla-ccm/archive/master.zip
```

Comment:
And here we use pip3? I'm confused now, do we need python 2 at all?

Reply:
I think `pip3` just refers to the pip module of the intended python version.